pina/geometry/triangle.py

```diff
@@ -0,0 +1,226 @@
+import torch
+import numpy as np
```

Comment:
remove numpy dependency, use only torch.

Reply:
In the most recent commit, numpy dependencies have all been replaced with torch operations.